lib/style.js

```diff
@@ -61,7 +58,8 @@ const parseRule = (ruleNode, dynamic) => {
         }
       });
       rules.push({
-        specificity: specificity(node),
+        // @ts-ignore internal api
+        specificity: syntax.specificity(node),
```

Comment:
The ts-ignore will become unnecessary if #1910 is merged. Same in `plugins/inlineStyles.js`.